app/forms/Constants.scala

```diff
@@ -17,7 +17,7 @@
 package forms
 
 object Constants {
-  lazy val maxNameLength: Int                       = 70
+  lazy val maxNameLength: Int                       = 35
```

Comment:
Is this used anywhere else? Might affect post-transition?

Reply:
Nope, only appears in the controller for Name I have created. Guessing this was created when we created the repo :)